iron-overlay-behavior.html

```diff
@@ -245,24 +245,28 @@
       // Used for wrapping the focus on TAB / Shift+TAB.
       this.__firstFocusableNode = this.__lastFocusableNode = null;
       // Used for requestAnimationFrame when opened changes.
-      this.__openChangedAsync = null;
+      this.__openedChangedRAF = null;
       // Used for requestAnimationFrame when iron-resize is fired.
-      this.__onIronResizeAsync = null;
+      this.__ironResizeRAF = null;
+      // Focused node before overlay gets opened. Can be restored on close.
+      this.__restoreFocusNode = null;
       this._ensureSetup();
     },
 
     attached: function() {
       // Call _openedChanged here so that position can be computed correctly.
       if (this.opened) {
-        this._openedChanged();
+        this._openedChanged(this.opened);
       }
       this._observer = Polymer.dom(this).observeNodes(this._onNodesChange);
     },
 
     detached: function() {
       Polymer.dom(this).unobserveNodes(this._observer);
       this._observer = null;
-      this.opened = false;
+      this.__openedChangedRAF && window.cancelAnimationFrame(this.__openedChangedRAF);
```

Comment:
is the compiler going to compile this away because it's not being assigned to anything? Is an if statement better?